doc/api/crypto.md

```diff
@@ -1365,6 +1365,13 @@ The `key` is the raw key used by the `algorithm` and `iv` is an
 [Buffers][`Buffer`], `TypedArray`, or `DataView`s. If the cipher does not need
 an initialization vector, `iv` may be `null`.
 
+Initialization vectors should be unpredictable; ideally, they will be
+cryptographically random. They do not have to be secret: IVs are typically just
+added to ciphertext messages in plaintext. It may sound contradictory that
```

Comment:
Maybe don't say 'plaintext' here, that has a specific meaning in crypto speak.

Reply:
Okay. I meant, it's sent "in the plain", or more precisely, unencrypted. I'll substitute "in plaintext" with "unencrypted". Is that better?